From cc187c04400bdddf55b51eb57da476b1a7de2c6c Mon Sep 17 00:00:00 2001
From: Tsuyoshi Ozawa <ozawa@apache.org>
Date: Sat, 5 Sep 2015 11:21:03 +0900
Subject: [PATCH 1697/2748] MAPREDUCE-6442. Stack trace is missing when error
 occurs in client protocol provider's constructor
 Contributed by Chang Li.

(cherry picked from commit 9b685773ec1031e90cc2b8aedb42670721757e22)
(cherry picked from commit 8bf536201400aaee1970428c87a117150308536f)

Change-Id: I1613947a18d23165fd44e57c5c69a65ac9af4251
---
 .../java/org/apache/hadoop/mapreduce/Cluster.java  |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/Cluster.java b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/Cluster.java
index 2fcc046..6cd2d12 100644
--- a/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/Cluster.java
+++ b/hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-core/src/main/java/org/apache/hadoop/mapreduce/Cluster.java
@@ -111,7 +111,7 @@ private void initialize(InetSocketAddress jobTrackAddr, Configuration conf)
         } 
         catch (Exception e) {
           LOG.info("Failed to use " + provider.getClass().getName()
-              + " due to error: " + e.getMessage());
+              + " due to error: ", e);
         }
       }
     }
-- 
1.7.9.5

